package-lock.json

```diff
@@ -143,7 +143,7 @@
         "string-width": {
           "version": "2.1.1",
           "resolved": "https://registry.npmjs.org/string-width/-/string-width-2.1.1.tgz",
-          "integrity": "sha512-nOqH59deCq9SRHlxq1Aw85Jnt4w6KvLKqWVik6oA9ZklXLNIOlqg4F2yrT1MVaTjAqvVwdfeZ7w7aCvJD7ugkw==",
+          "integrity": "sha1-q5Pyeo3BPSjKyBXEYhQ6bZASrp4=",
```

Comment:
Hmm. Which npm version are you using?

Reply:
I'm using 6.2.0. I should probably upgrade. What are you using?

Reply:
![image](https://user-images.githubusercontent.com/6675409/49385715-0fa53480-f6ec-11e8-9077-bfa59db76d47.png)

yeah, upgrading would be best.